Sources/TuistGraph/Models/ProjectOptions.swift

```diff
@@ -26,7 +26,25 @@ extension Project {
             textSettings: TextSettings
         ) {
             self.automaticSchemesOptions = automaticSchemesOptions
-            self.disableBundleAccessors = disableBundleAccessors
+            if disableBundleAccessors {
+                bundleAccessorsOptions = .disabled
+            } else {
+                bundleAccessorsOptions = .enabled(includeObjcAccessor: true)
+            }
+            self.disableShowEnvironmentVarsInScriptPhases = disableShowEnvironmentVarsInScriptPhases
+            self.disableSynthesizedResourceAccessors = disableSynthesizedResourceAccessors
+            self.textSettings = textSettings
+        }
+
+        public init(
```

Comment:
I'm not sure if we want to deprecate old initializers, I had to add an additional one to keep backward compatibility and allow compilation (default values in the functions caused a compilation issue)